src/content/self-hosted/helm-configuration.mdx

```diff
@@ -483,6 +483,39 @@ regcredsManager:
       memory: 800Mi
 ```
 
+### resourceManager
+
+Use this section to configure the [Resource Manager](admin/resource-manager.mdx)
+
+- `enabled`: When set to `true`, the Resource Manager is installed. When Resource Manager is installed, Okteto creates a CronJob responsible for computing the cpu and memory recommendations for your pods. It defaults to `false`.
```

Comment:
Minor: could you sort the fields in alphabetical order?

Reply:
@pchico83 shall we sort them also in `values.yaml`?